build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -41,7 +41,7 @@
         <Inputs>@(CompileStageInputs)</Inputs>
         <StageDirectory>$(Stage2Directory)</StageDirectory>
         <StageSymbolsDirectory>$(Stage2SymbolsDirectory)</StageSymbolsDirectory>
-        <DotnetDir>$(Stage1Directory)</DotnetDir>
+        <DotnetDir>$(Stage0Directory)</DotnetDir>
```

Comment:
Can we skip stage1 all together now?  At least comment it out so we don't need to run the same thing twice.